examples/resources/hpegl_vmaas_load_balancer_monitor/nsx_t_lb_tcp_monitor.tf

```diff
@@ -0,0 +1,18 @@
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+# TCP Monitor
+resource "hpegl_vmaas_load_balancer_monitor" "tf_TCP_MONITOR" {
+  lb_id = data.hpegl_vmaas_load_balancer.tf_lb.id 
+  name  =  "tf_TCP_MONITOR"       
+  description  = "TCP_MONITOR create using tf"
+  type = "LBTcpMonitorProfile"
+  tcp_monitor {
+    fall_count = 3
+  interval =  5 
+  monitor_port = 80
+  rise_count = 3
+  timeout = 15
+  request_body = "request body data"
```

Comment:
if tcp and udp has common params, why have them separet schema for them ?

Reply:
let it be different for each of tcp and udp so that in future if modify any fields will take it separate or else in the future needs the same rework again